.storybook/main.js

```diff
@@ -6,9 +6,12 @@ module.exports = {
   },
   typescript: {
     // also valid 'react-docgen-typescript' | false
-    reactDocgen: 'react-docgen',
+    reactDocgen: 'react-docgen-typescript',
```

Comment:
MMmmh on est surs de ca? de souvenir c'etait pour les args table

Reply:
We are migrating all files to typescript which makes args table non functional (checkout PR on rich select). It is required to go trough react-docgen-typescript